source/features/index.js

```diff
@@ -0,0 +1,181 @@
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+
+export const features = {
+	/* GENERAL */
+	keyboardShortcuts: {
+		id: 'feature-keyboard-shortcuts',
+		category: 'general',
+		label: 'Enable keyboard shortcut to toggle Night Mode (Alt + M)',
+		fn: require('./keyboard-shortcuts').default
+	},
+	preserveTextMessages: {
+		id: 'feature-preserve-text-messages',
+		category: 'general',
+		label: 'Preserve unsent text in the Messages modal when it closes',
+		fn: require('./preserve-text-messages').default
+	},
+	useSystemFont: {
+		id: 'feature-use-system-font',
+		category: 'general',
+		label: 'Use the system font',
+		init: true
```

Comment:
`init` => `isEnabled`

Reply:
Different things, I'm using init as in "automatically start this feature on the init() method"; any other better name?

Reply:
`runOnInit`?